packages/packages.csv

```diff
@@ -106,14 +107,14 @@ Microsoft.AspNetCore.SignalR.SqlServer,noship
 Microsoft.AspNetCore.StaticFiles,ship
 Microsoft.AspNetCore.TestHost,ship
 Microsoft.AspNetCore.Testing,noship
-Microsoft.AspNetCore.WebSockets.Client,ship
+Microsoft.AspNetCore.WebSockets.Client,noship
 Microsoft.AspNetCore.WebSockets.Protocol,ship
 Microsoft.AspNetCore.WebSockets.Server,ship
 Microsoft.AspNetCore.WebUtilities,ship
 Microsoft.Cci,ext
 Microsoft.CodeAnalysis.*,ext
 Microsoft.CSharp,ext
-Microsoft.Data.Sqlite,ship
+Microsoft.Data.Sqlite,ship-ext
```

Comment:
Splitting is also a nuanced question. See https://github.com/aspnet/Microsoft.Data.Sqlite/issues/237. Regarding signing, see https://github.com/aspnet/Coherence-Signed/pull/285